README.md

```diff
@@ -267,17 +268,46 @@ Stripe.product :primo do |product|
 end
 ```
 
-To upload your plans and coupons onto stripe.com, run:
+And Prices:
+
+```ruby
+Stripe.price :bronze do |price|
+  # Use an existing product id to prevent a new product from
+  # getting created
+  price.product_id = Stripe::Products::PRIMO.id
+  price.unit_amount = 999 # $9.99
+  price.interval = 'month'
```

Comment:
I think interval needs be to in a `recurring` hash

Reply:
👍 Good find!  Cleared up this one and the one just below.  I searched for 'interval' but didn't spot anything else that looked like it was still the old style.